### PR TITLE
Add active class to Like/Compare buttons

### PR DESCRIPTION
**Pull request by Ksz-K**
Some changes I made the second time due to unmerged to master status of previous branch/task

### CODE REVIEW

File: src/partials/50_section-products.html [hunk 1]
```diff
@@ -100,7 +100,9 @@ <h5>Aenean Ru Bristique</h5>
           <div class="actions">
             <div class="outlines">
               <a href="#" class="btn-outline"><i class="far fa-heart"></i></a>
-              <a href="#" class="btn-outline"><i class="fas fa-exchange-alt"></i></a>
+              <a href="#" class="btn-outline active"
+                ><i class="fas fa-exchange-alt"></i
```
Comment (Neonides) [inline src/partials/50_section-products.html]: Tu chyba zamknięcia weszły do drugiej lini.

File: src/partials/50_section-products.html [hunk 2]
```diff
@@ -127,8 +129,10 @@ <h5>Aenean Ru Bristique</h5>
           <div class="line"></div>
           <div class="actions">
             <div class="outlines">
-              <a href="#" class="btn-outline"><i class="far fa-heart"></i></a>
-              <a href="#" class="btn-outline"><i class="fas fa-exchange-alt"></i></a>
+              <a href="#" class="btn-outline active"><i class="far fa-heart"></i></a>
+              <a href="#" class="btn-outline active"
+                ><i class="fas fa-exchange-alt"></i
```
Comment (Neonides) [inline src/partials/50_section-products.html]: j.w

Comment (Ksz-K) [reply src/partials/50_section-products.html]: Done

**Comment by matwjablonski** [comment]
please resolve conflicts

**Pull request merged**